src/hooks/useServerApi.ts

```diff
@@ -117,25 +128,23 @@ export const useServerApi = () => {
       }
       headers.Authorization = `Bearer ${token}`;
     }
+    const finalHeader = setHeaderForImpersonation(headers, impersonationData);
 
-    headers['x-accept-versions'] = version ? version : '1.0.3';
-
-    const updatedHeaders = setHeaderForImpersonation(
-      headers,
-      impersonationData
-    );
+    const baseUrl = process.env.API_ENDPOINT;
+    if (!baseUrl)
+      throw new Error(
+        'API_ENDPOINT is not defined in your environment variables.'
+      );
 
-    const fullUrl = isAbsoluteUrl(url)
-      ? url
-      : `${process.env.API_ENDPOINT}${url}`;
+    const fullUrl = isAbsoluteUrl(url) ? url : `${baseUrl}${url}`;
```

Comment:
Move to apiClient

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2455/commits/7bcd95fb4138015b860f687f0d827115ee94c76f